src/app/lib/utilities/srcSet/index.js

```diff
@@ -1,6 +1,24 @@
 import buildIChefUrl from '#lib/utilities/ichefURL';
 
 const DEFAULT_RESOLUTIONS = [240, 320, 480, 624, 800];
+const PIPS_RESOLUTIONS = [
```

Comment:
that's a lot of resolutions. where did you get these resolution requirements?

is this width / height? it would be better to work out on dimension using an aspect ratio. I'm assuming this is how DEFAULT_RESOLUTIONS works

Reply:
I belive these are the requirements for the SMP built-in image placeholder. Currently on live canonical we have the following resolutins under
 srcset: 
 <img width="1008" alt="Screenshot 2022-01-10 at 11 03 21" src="https://user-images.githubusercontent.com/90621252/148755782-69d88879-c7ea-404a-89b1-97f320200cf8.png">

Reply:
I just discovered a way to create a working link using only the width. I will make this change now 👍

Reply:
> Can you also please remove the amp-img placeholder? I don't think this is needed

Yes I can do that 👍